server/src/main/java/com/distkv/server/metaserver/server/bean/BaseResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.distkv.server.metaserver.server.bean;
+
+import java.io.Serializable;
+
+public class BaseResponse implements Serializable {
```

Comment:
This should be abstract right?

Reply:
I don't think it's necessary to treat it as an abstract class. I think inheritance makes sense.